codegen/arrai/svc_types.arrai

```diff
@@ -69,6 +69,37 @@ let orderedTypes = \types
                         >> $`${go.name(.@item('name').s)} ${go.type(.@item('type'))}`
                     ::\i}
                 }
+
+                ${
+                let opParams = ep('restParams')?('queryParam')?.a:{} where $`${sysl.isPtr(.@item('type'))}`;
+                cond {opParams: $`
```

Comment:
This will error if it doesn't match. `opParams` is treated as a name to bind, not a value to test against and fails if the existing binding is different. Use `(opParams)` instead.

Reply:
This change breaks with

Reply:
I think this is fixed in the current version of arr.ai.